dtonator/src/main/java/com/bizo/dtonator/GenerateEnum.java

```diff
@@ -44,6 +52,29 @@ private void addEnumValues() {
     }
   }
 
+  private void addEnumGetters() {
+    try {
+      final Class<?> clazz = Class.forName(dto.getDomainType());
+      for (final Field field : clazz.getDeclaredFields()) {
+        if (!field.isEnumConstant() && !field.isSynthetic()) {
+          final GMethod getter = gc.getMethod("get{}", capitalize(field.getName())).returnType(field.getType());
```

Comment:
By initially looking at the fields, and calling "get<FieldName>", we're assuming that the getter has a field, and that their field/getter names match.

Granted, they should, but it might be nice to try and iterate over all "getXxx" methods, e.g. ignore the fields, that way we know for sure the getter exists, and then might also be able to map "getFoo" methods that have tiny bits of logic in them and aren't derived solely from fields.